ts/Series/AreaRange/AreaRangeSeries.ts

```diff
@@ -560,14 +560,32 @@ class AreaRangeSeries extends AreaSeries {
 
     public drawPoints(): void {
         const series = this,
-            pointLength = series.points.length;
+            pointLength = series.points.length,
+            seriesOptionsMarker = series.options.marker,
+            seriesDefaultSymbol = series.symbol;
 
         let i: number,
             point: AreaRangePoint;
 
+        if (series.options.lowMarker) {
+            const { options: { marker, lowMarker } } = series;
+
+            series.options.marker = merge(marker, lowMarker);
+
+            if (lowMarker.symbol) {
+                series.symbol = lowMarker.symbol;
+            }
+        }
```

Comment:
It seams to me that this code block, and the restore block below, are repeated between the AreaRangePoint.ts and AreaRangeSeries.ts files. In adherence with the [DRY principle](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself), we should refactor it out to a function. The function should be placed on the AreaRangeSeries class since it handles stuff on the series level.

One way to resolve it is to do the two operations in the same function. So the first time it runs (without arguments), it returns the original settings for marker and symbol. The second time it is called with the original settings as an argument, and then those properties are restored.